packages/rollup/test/integration/far/a/tsconfig.json

```diff
@@ -5,5 +5,11 @@
     "target": "es2015",
     "module": "esnext",
     "moduleResolution": "node"
-  }
+  },
+  // For Windows, which is not sand-boxed, we need to limit
```

Comment:
Because the default is `**/*.ts` which includes subdirectories?

Reply:
Yes